ui/src/Components/Dashboard/Table/TableHeader/SortCursor.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { TABLE } from "../../../../Constant/constant";
+
+export default function SortCursor({ header, handleSorting }) {
+  const [currentClickedCursor, setCurrentClickedCursor] = useState(null);
+
+  const onCursorClick = (clickedCursorType, headerName) => {
+    setCurrentClickedCursor(clickedCursorType);
+    handleSorting(clickedCursorType, headerName);
+  };
+
+  return (
+    <div>
+      <div
+        className={`up-arrow ${
+          currentClickedCursor === TABLE.ASCENDING ? "clicked-u" : null
+        }`}
+        title={TABLE.ASCENDING}
+        onClick={() => {
+          onCursorClick(TABLE.ASCENDING, header);
+        }}
+      ></div>
+      <div
+        className={`down-arrow ${
+          currentClickedCursor === TABLE.DESCENDING ? "clicked-d" : null
```

Comment:
`Clicked-d` needs to move to constant

Reply:
@Neha clicked-d is a css className that i'm adding in conditionally.